src/pages/api/github-load-notes.ts

```diff
@@ -0,0 +1,89 @@
+import {NextApiRequest, NextApiResponse} from 'next'
+import mdx from '@mdx-js/mdx'
+import VFile from 'vfile'
+import visit from 'unist-util-visit'
+import fetch from 'isomorphic-fetch'
+import toMarkdown from 'mdast-util-to-markdown'
+
+const loadGithubNotes = async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method === 'GET' && req.query.url) {
+    const test = await loadNotesFromUrl(req.query.url as string)
+
+    res.status(200).json(test)
+  } else {
+    res.status(200).end()
+  }
+}
+
+export default loadGithubNotes
+
+export async function loadNotesFromUrl(url: string) {
```

Comment:
this loads mdx from a url and parses it looking for timestamp components which creates JS objects that are used to make a webvtt.

it's currently very fragile! works though